programmers/난이도별/level01.서울에서_김서방_찾기/HyeonJeong.py

```diff
@@ -0,0 +1,7 @@
+def solution(seoul):
+    for m, n in enumerate(seoul):
+        if "Kim" == n:
+            return ("김서방은 %d에 있다" %m)
+            # return "김서방은 {}에 있다".format(m)
```

Comment:
```index(x)``` 함수는 리스트 자료에서 x의 위치를 반환해줍니다! 이를 사용하면 코드를 한 줄로 줄일 수 있습니다.

x는 실수, 문자열, 불린 값(True, Flase), 리스트, 튜플, 딕셔너리, 집합 모두 들어갈 수 있습니다. 

제가 생각하기에 ```index()```함수를 사용하실때 주의하실점 두가지는
1. 리스트에 x값이 없다면 에러를 띄운다.
2. 중복된 값이 있을 경우 가장 먼저 나오는 x의 자리만을 return한다.

정도입니다.

```suggestion
    return '김서방은 %d에 있다'%seoul.index('Kim')
```

Reply:
리스트의 값이 한번 존재하거나 중복 값 중 우선적으로 나오는 값을 사용할 때 유용하게 사용할 수 있겠네요! 감사합니다.